src/code.cloudfoundry.org/go.mod

```diff
@@ -63,20 +63,16 @@ require (
 )
 
 require (
-	code.cloudfoundry.org/consuladapter v0.0.0-20210615194356-31457193d2fd // indirect
 	code.cloudfoundry.org/diego-logging-client v0.0.0-20210622170659-8861ae5ba2ed // indirect
 	code.cloudfoundry.org/durationjson v0.0.0-20210615172401-3a89d41c90da // indirect
 	code.cloudfoundry.org/go-diodes v0.0.0-20190809170250-f77fb823c7ee // indirect
 	code.cloudfoundry.org/go-loggregator/v8 v8.0.5 // indirect
 	code.cloudfoundry.org/inigo v0.0.0-20210615140442-4bdc4f6e44d5 // indirect
-	code.cloudfoundry.org/rep v0.1441.2 // indirect
+	code.cloudfoundry.org/lager v1.1.1-0.20230321195817-3d52f427a2d2 // indirect
```

Comment:
@geofffranks I think this PR might be premature needs one more change. 

There's still a reference to lager v1. It took a bit of digging, but the only reference I could find to `code.cloudfoundry.org/lager` without a `v3` namespace was in the [BBS packages that get vendored in](https://github.com/cloudfoundry/routing-release/blob/ginkgo-v2/src/code.cloudfoundry.org/vendor/code.cloudfoundry.org/bbs/models/task.go#L10).

Luckily, all it took was a quick update to BBS to get rid of the reference to lager v1:
```
go get code.cloudfoundry.org/bbs
go mod tidy
go mod vendor
```

I don't have push access to this branch, so I'll let you fix it.

Reply:
Fixed